drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/auditlog/AuditLogEntryCellHelperTest.java

```diff
@@ -546,66 +547,81 @@ private String getNonNullValue(Object diffValue, boolean constraintValueType) {
     }
 
     // this might be useful in the helper itself..?
-    private String getDisplayableFieldValue( String field, String optionalParameter, boolean WIParamValueOnly ) {
-        switch ( field ) {
-            case DTColumnConfig52.FIELD_HEADER: return GuidedDecisionTableConstants.INSTANCE.ColumnHeader();
-            case DTColumnConfig52.FIELD_HIDE_COLUMN : return GuidedDecisionTableConstants.INSTANCE.HideThisColumn();
-            case DTColumnConfig52.FIELD_DEFAULT_VALUE : return GuidedDecisionTableConstants.INSTANCE.DefaultValue();
-            case MetadataCol52.FIELD_METADATA : return GuidedDecisionTableConstants.INSTANCE.Metadata1();
-            case AttributeCol52.FIELD_REVERSE_ORDER : return GuidedDecisionTableConstants.INSTANCE.ReverseOrder();
-            case AttributeCol52.FIELD_USE_ROW_NUMBER : return GuidedDecisionTableConstants.INSTANCE.UseRowNumber();
-            case Pattern52.FIELD_ENTRY_POINT_NAME : return GuidedDecisionTableConstants.INSTANCE.DTLabelFromEntryPoint();
-            case ConditionCol52.FIELD_BINDING : return GuidedDecisionTableConstants.INSTANCE.Binding();
-            case ConditionCol52.FIELD_CONSTRAINT_VALUE_TYPE : return GuidedDecisionTableConstants.INSTANCE.CalculationType();
-            case ConditionCol52.FIELD_OPERATOR : return GuidedDecisionTableConstants.INSTANCE.Operator();
-            case ConditionCol52.FIELD_FIELD_TYPE :
-            case ActionSetFieldCol52.FIELD_TYPE : return GuidedDecisionTableConstants.INSTANCE.FieldType();
-            case ActionSetFieldCol52.FIELD_UPDATE : return GuidedDecisionTableConstants.INSTANCE.UpdateEngineWithChanges();
-            case ActionInsertFactCol52.FIELD_IS_INSERT_LOGICAL : return GuidedDecisionTableConstants.INSTANCE.LogicallyInsert();
-            case ActionWorkItemCol52.FIELD_WORKITEM_DEFINITION_NAME : return GuidedDecisionTableConstants.INSTANCE.DecisionTableAuditLogWorkItemName();
-            case ActionWorkItemCol52.FIELD_WORKITEM_DEFINITION_PARAMETER_NAME : return GuidedDecisionTableConstants.INSTANCE.DecisionTableAuditLogWorkItemParameterName();
-            case ActionWorkItemCol52.FIELD_WORKITEM_DEFINITION_PARAMETER_VALUE : {
-                if ( WIParamValueOnly ) {
-                    return GuidedDecisionTableConstants.INSTANCE.DecisionTableAuditLogWorkItemParameterValueOnly0( optionalParameter );
+    private String getDisplayableFieldValue(String field, String optionalParameter, boolean WIParamValueOnly) {
+        switch (field) {
+            case DTColumnConfig52.FIELD_HEADER:
```

Comment:
Suggest to change it to map since that does not have default

Reply:
If possible, I suggest to add a default behavior for this switch.